core/src/main/java/com/digitalpebble/stormcrawler/bolt/SimpleFetcherBolt.java

```diff
@@ -402,16 +402,20 @@ public void execute(Tuple input) {
                 message = "";
 
             // common exceptions for which we log only a short message
-            if (exece.getCause() instanceof java.util.concurrent.TimeoutException)
+            if (exece.getCause() instanceof java.util.concurrent.TimeoutException) {
                 LOG.error("Socket timeout fetching {}", urlString);
-            else if (message.contains(" timed out"))
+                message = "Socket timeout fetching";
+            } else if (message.contains(" timed out")) {
```

Comment:
Yeah, definitely. To be honest I was so busy putting in the message delta I didn't look at the original flow.